catalog/core/catalog-core-federationstrategy/src/test/java/ddf/catalog/federation/impl/FederationStrategyTest.java

```diff
@@ -498,6 +499,7 @@ public void testFederateOneSourceOffsetTwoPageSizeTwo() throws Exception {
         LOGGER.debug("testFederate_OneSource_OffsetTwo_PageSizeTwo()");
         // Test Setup
         Query mockQuery = mock(QueryImpl.class);
+        when(mockQuery.getTimeoutMillis()).thenReturn(LONG_TIMEOUT);
```

Comment:
❓ Since we use the same return value for all `getTimeoutMillis()` calls, should we just create the mock `Query.class` in the `setup()` method and put the expectation there instead of repeating it in 4 different places?